english/malaria/malaria.md

```diff
@@ -5,7 +5,7 @@ around to take a look at the marshmallow hand and in a flash you are
 enveloped in a sea of marshmallow. Wherever you see, there is nothing but 
 marshmallow. You flail and trash but to no avail. You cannot draw breath. 
 You are suffocating! It starts getting dark to your eyes. 
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.